### PR TITLE
[DOCU-1798]Adds changelog entries for the 2.2.1.5, 2.3.3.4, 2.4.1.3, and 2.5.0.2 hot fix releases

### PR DESCRIPTION
**Pull request by HCloward**
### Summary
Main update is to inform users about the Dev Portal template issue being fixed by the hot fixes - [FTI-2828](https://konghq.atlassian.net/browse/FTI-2828), and for 2.5.0.2 a fix for the Mocking plugin was also included - [FTI-2582](https://konghq.atlassian.net/browse/FTI-2582)
### Reason
[DOCU-1798](https://konghq.atlassian.net/browse/DOCU-1798)
### Testing
2.2.1.5
2.3.3.4
2.4.1.2
2.5.0.2

### CODE REVIEW

File: app/enterprise/changelog.md [hunk 1]
```diff
@@ -4,6 +4,20 @@ no_search: true
...
 ---
 
+## 2.5.0.2
+**Release Date** 2021/09/02
+
+### Fixes
+
+#### Enterprise
+- This release fixes a regression in the Kong Dev Portal templates that removed dynamic menu navigation and other improvements
+from portals created in Kong Gateway v2.2.1.4.
```
Comment (tyler-ball) [inline app/enterprise/changelog.md:14]:
```suggestion
from portals created in Kong Gateway v2.5.0.1.
```

File: app/enterprise/changelog.md [hunk 2]
```diff
@@ -497,6 +511,15 @@ no_version: true
...
 
 
+## 2.4.1.3
+**Release Date** 2021/09/02
+
+### Fixes
+
+#### Enterprise
+- This release fixes a regression in the Kong Dev Portal templates that removed dynamic menu navigation and other improvements
+from portals created in Kong Gateway v2.2.1.4.
```
Comment (tyler-ball) [inline app/enterprise/changelog.md:521]:
```suggestion
from portals created in Kong Gateway v2.4.1.2.
```

File: app/enterprise/changelog.md [hunk 3]
```diff
@@ -937,6 +960,15 @@ keep-alive connections. [7102](https://github.com/Kong/kong/pull/7102)
...
 
 
+## 2.3.3.4
+**Release Date** 2021/09/02
+
+### Fixes
+
+#### Enterprise
+- This release fixes a regression in the Kong Dev Portal templates that removed dynamic menu navigation and other improvements
+from portals created in Kong Gateway v2.2.1.4.
```
Comment (tyler-ball) [inline app/enterprise/changelog.md:970]:
```suggestion
from portals created in Kong Gateway v2.3.3.3.
```

**Review by tyler-ball — APPROVED** [review]
Fixed some copy-pasta mistakes I made when creating the changelogs in Jira. Otherwise this LGTM!

**Review by lena-larionova — APPROVED** [review]

**Pull request merged**